conf/config.yaml

```diff
@@ -21,7 +21,7 @@ apisix:
   enable_admin_cors: true         # Admin API support CORS response headers.
   enable_debug: false
   enable_dev_mode: false          # Sets nginx worker_processes to 1 if set to true
-  enable_reuseport: true          # Enable nginx SO_REUSEPORT switch if set to true.
+  enable_reuseport: false          # Enable nginx SO_REUSEPORT switch if set to true.
```

Comment:
default value `true` is better

Reply:
setting enable_reuseport false to verify that test cases are working.

Reply:
For testing, you can write a separate script to modify `config.yaml`.